src/client/Microsoft.Identity.Client/Internal/RequestContext.cs

```diff
@@ -24,6 +25,8 @@ internal class RequestContext
 
         public CancellationToken UserCancellationToken { get; }
 
+        public Dictionary<string, object> CacheDetails { get; } = new Dictionary<string, object>();
```

Comment:
Can we name this `TelemetryDatapoints` or something more general purpose?

Reply:
When you create the dictionary, specify the culture, so that key operations are case insensitive. https://stackoverflow.com/questions/13988643/case-insensitive-dictionary-with-string-key-type-in-c-sharp